packages/vuetify/src/components/VList/VListGroup.sass

```diff
@@ -50,13 +50,23 @@
   .v-list-item__icon
     &:first-child
       margin-right: 16px
+      +rtl()
```

Comment:
Please use here (and in other places) ltr() function: https://github.com/vuetifyjs/vuetify/blob/b76e5e52dee99cf16c0aac3e5db5f04dc1091f5e/packages/vuetify/src/components/VTabs/VTabs.sass#L52-L57

Reply:
check it now :)